views.py

```diff
@@ -654,7 +657,9 @@ def edit_annotation_base(request, image_pid, anno_pid, redirect_url):
         context_data.update(get_bound_edit_forms(annotation, AnnotationForm, PersonFormSet, InscriptionFormSet))
 
     image_link = 'https://%s/viewers/image/zoom/%s' % (BDR_SERVER, image_pid)
-    context_data.update({'image_link': image_link})
+    person_helper = PersonFormHelper()
```

Comment:
Seems like the helpers should be set within the form.
And shouldn't have to be initialized explicitly.
http://django-crispy-forms.readthedocs.org/en/d-0/tags.html#rendering-several-forms-with-helpers